### DISCUSSION TITLE
Рубрика 100 советов в зоне?

### DISCUSSION THREAD

**Discussion by viktar83** [OGSR/OGSR-Engine#291] [Ideas]
Здравствуйте.
Планируете ли задействовать данную функцию на загрузочном экране?

Цитаты из АМК-форума [https://www.amk-team.ru/forum/topic/14065-ogsr-mod-pravki-i-modifikacii/page/20/?tab=comments#comment-1287461](url)

> Можно отдельно сделать рандомные надписи через "100 советов по выживанию в зоне".

> Добавлено KRodin, 30 Сентября 2019. Вообще-то нельзя. Я их в движке не доделывал, за ненадобностью.

Эх, а надобность-то есть.

Можно, конечно, реализовать как в НС2016 OGSR через костыли (через текстуры), но это неудобно и некрасиво.

**Comment by NikitaNikson** [comment]
согласен
было бы очень здорово, если подвезёте подобную фичу =)

**Comment by xrSimpAI** [comment]
Можно подумать над этим в будущем.

**Comment by viktar83** [comment]
Спасибо. Было бы неплохо.
Переделал соляночную реализацию под текстовый вариант, но всё равно это какой-то костыль.

**Comment by xrSimpAI** [comment]
Понятное дело, они ж эту реализацию под ванильный движок делали, там по другому нельзя было.
